### PR TITLE
changed default enclosure tag to require $$

### PR DESCRIPTION
**Pull request by vincegando**
Changed default Mathjax enclosure tag from "$" to "$$" as referenced in #1096 

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [ ] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
This looks great; have you been able to confirm that it works in your local setup? Just create a note with the specified tags inside `<p></p>` tags.

**Comment by vincegando** [comment]
No, I haven't gotten it to work in my local setup. However, I was able get it up and running on the Cloud9 setup.

What do you mean by a note? In which file?

**Comment by ananyo2012** [comment]
@vincegando Please hold on to this PR a bit. I have proposed some fixes for this in #1096. 
Also what Jeff meant by a note is a research note which you can create from your dashboard from the Share ypur work button. Just post a research note to try that this works.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -20,7 +20,7 @@
       src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
     </script>
     <script type="text/x-mathjax-config">
-      MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
+      MathJax.Hub.Config({tex2jax: {inlineMath: [['$$','$$'], ['\\(','\\)']]}});
```
Comment (ananyo2012) [inline app/views/layouts/application.html.erb]: Sorry for the confusion there. But just removing this line will work.

**Comment by jywarren** [comment]
Ananyo I'm happy to defer to your judgement on this and when it's ready to
be merged -- thanks! Traveling at the moment.

**Comment by jywarren** [comment]
Hi, all - sorry, i'm traveling at the moment and less able to respond quickly. But this looks great, merging and thanks @vincegando !!

**Pull request merged**